docs/install-mlrun-ce/mlrun-ce-development-notes.md

```diff
@@ -0,0 +1,79 @@
+# MLRun CE Development Notes
+
+This page contains development notes for configure your developemt system.
+
+**In this section**
+
```

Comment:
- [Change the deployment and jobs default PVC](#change-the-deployment-and-jobs-default-pvc)